Classes/Emogrifier.php

```diff
@@ -308,7 +310,9 @@ public function emogrifyBodyContent()
         }
 
         $xmlDocument = $this->createXmlDocument();
+        set_error_handler([$this, 'handleXpathError'], E_WARNING);
         $this->process($xmlDocument);
+        restore_error_handler();
```

Comment:
Maybe we can move handler code to `process()` or add `processWithErrorHandling()` method